stack/modules/d7ap_fs/CMakeLists.txt

```diff
@@ -22,7 +22,13 @@
 #See cmake/module_macros.cmake for more information
 
 MODULE_OPTION(${MODULE_PREFIX}_USE_DEFAULT_SYSTEMFILES "Use the default D7AP systemfiles values" TRUE)
-MODULE_HEADER_DEFINE(BOOL ${MODULE_PREFIX}_USE_DEFAULT_SYSTEMFILES)
+MODULE_HEADER_DEFINE()
```

Comment:
can be removed

Reply:
MODULE_HEADER_DEFINE() probably as well since it is on line 28?

Reply:
on line 28 can remain since this defines the headers but this one is empty